sys/riscv/riscv/locore.S

```diff
@@ -546,7 +546,7 @@ initstack_end:
  * to execute.
  */
 #define	SIGCODE(sym)						\
-	.text; .globl sym; .type sym,@object; .align 4; sym:
+	.text; .globl sym; .type sym,@object; .align 4; sym: .cfi_startproc
```

Comment:
This one we will just squash down into the relevant change when we merge it up to dev.

Reply:
Alright